explainer.md

```diff
@@ -1,22 +1,23 @@
-## Explainer
+# Explainer
 
-Author: Eric Willigers &lt;<ericwilligers@chromium.org>&gt;<br>
-Author: Jay Harris &lt;<harrisjay@chromium.org>&gt;<br>
-Author: Raymes Khoury &lt;<raymes@chromium.org>&gt;
+Author: 
+* Darwin Huang &lt;<huangdarwin@chromium.org>&gt;
```

Comment:
I would like it if you kept the original authors names here, even if you're substantially rewriting the doc.

I see you've moved them to the bottom as "former editors", but this isn't really like a spec where you have Editors / Former Editors, rather it's just a list of people who've worked on this doc, as you would have in a Google Doc.

Reply:
I moved these author names down because of https://github.com/WICG/file-handling/pull/46#discussion_r541413788.

I've updated the author section to avoid downplaying the original authors' contributions, while still being clear that I'm the current owner / should be responsible for future questions/changes. Hopefully this seems better, but I'm still uncertain of the best wording here. Please let me know of a better wording and I can certainly update this in a follow-up PR.